server_admin/topics/users/user-profile.adoc

```diff
@@ -0,0 +1,306 @@
+
+
+=== User Profile
+
+In {project_name} a user is associated with a set of attributes. These attributes are used to better describe and identify users within Keycloak as well as to pass over additional information to applications about them.
```

Comment:
"...within {project_name}..." perhaps?

Reply:
Fixed and replaced in a few other places too.